src/v/kafka/server/group_data_parser.h

```diff
@@ -175,43 +199,68 @@ class group_data_parser {
           fence_version,
           group::fence_control_record_version);
     }
+
     ss::future<> handle_raft_data(model::record_batch b) {
         return static_cast<Base*>(this)->handle_raft_data(std::move(b));
     }
     ss::future<> handle_tx_offsets(
       model::record_batch_header header,
       kafka::group_tx::offsets_metadata data) {
+        if (is_group_blocked_verbose(data.group_id, "tx offsets")) {
```

Comment:
I'm wondering if we can bump the log level here.. a new transaction after a group block is a bug for sure?